### ISSUE TITLE
Enhance accessibility

### ISSUE THREAD

**Issue by mikaoelitiana** [wix/react-native-calendars#373]
## Description

We use this library in our app and we received suggestion to enhance accessibility on calendar header by adding accessibilityTraits='header' to header element.

## Expected Behavior

Calendar header text should have 'header' traits (accessibilityTraits='header')

## Observed Behavior

Calendar header has no accessibility trait.

## Environment

Please run these commands in the project folder and fill in their results:

* `npm ls react-native-calendars`: react-native-calendars@1.17.0
* `npm ls react-native`: react-native@0.51.0

Also specify:

1. Phone/emulator/simulator & version: iPhone 6, iOS 11.2 (simulator)

## Reproducible Demo

Launch app on emulator and then start 'Accessibility Inspector' and point to calendar header. In Basic info, the type is text. It should be AXHeading.

![capture d ecran 2018-03-13 a 11 56 17](https://user-images.githubusercontent.com/674667/37337825-bd090ab4-26b5-11e8-928f-f19848feff6e.png)
